fbinternal/cloud/go/services/fbinternal/fbinternal/main.go

```diff
@@ -25,16 +25,27 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	defaultCategoryID = "magma"
+)
+
 func main() {
 	srv, err := service.NewOrchestratorService(fbinternal.ModuleName, fbinternal_service.ServiceName)
 	if err != nil {
 		glog.Fatalf("Error creating orc8r service for fbinternal: %s", err)
 	}
-
+	var categoryID string
+	categoryIDEnv := os.Getenv("FACEBOOK_APP_CATEGORY_ID")
```

Comment:
I think we have a utility function in the codebase to get env with a default, check the DB stuff.